--- a/config/delegates.xml.in
+++ b/config/delegates.xml.in
@@ -67,27 +67,27 @@
   <delegate decode="https:decode" command="@SHELL_QUOTE@@WWWDecodeDelegate@@SHELL_QUOTE@ -s -L -o @SHELL_QUOTE@%u.dat@SHELL_QUOTE@ @SHELL_QUOTE@https:%M@SHELL_QUOTE@"/>
   <delegate decode="doc" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; @MVDelegate@ @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
   <delegate decode="docx" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; @MVDelegate@ @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
-  <delegate decode="dng:decode" command="@MKDIRDelegate@ @SHELL_QUOTE@%u-config@SHELL_QUOTE@;@MVDelegate@ @SHELL_QUOTE@%i@SHELL_QUOTE@ @SHELL_QUOTE@%i.%e@SHELL_QUOTE@; @SHELL_QUOTE@@DNGDecodeDelegate@@SHELL_QUOTE@ --import @SHELL_QUOTE@%i.%e@SHELL_QUOTE@ @SHELL_QUOTE@%u.tif@SHELL_QUOTE@ --core --configdir @SHELL_QUOTE@%u-config@SHELL_QUOTE@ &gt; @SHELL_QUOTE@%u@SHELL_QUOTE@;@RMDelegate@ -r @SHELL_QUOTE@%u-config@SHELL_QUOTE@"/>
+  <delegate decode="dng:decode" command="@MKDIRDelegate@ @SHELL_QUOTE@%u-config@SHELL_QUOTE@;@MVDelegate@ @SHELL_QUOTE@%i@SHELL_QUOTE@ @SHELL_QUOTE@%i.%e@SHELL_QUOTE@; @SHELL_QUOTE@@DNGDecodeDelegate@@SHELL_QUOTE@ --import @SHELL_QUOTE@%i.%e@SHELL_QUOTE@ @SHELL_QUOTE@%u.tif@SHELL_QUOTE@ --core --configdir @SHELL_QUOTE@%u-config@SHELL_QUOTE@ &gt; @SHELL_QUOTE@%u@SHELL_QUOTE@;/opt/bin/rm -r @SHELL_QUOTE@%u-config@SHELL_QUOTE@"/>
   <delegate decode="dot" command="@SHELL_QUOTE@@GVCDecodeDelegate@@SHELL_QUOTE@ -Tsvg @SHELL_QUOTE@%i@SHELL_QUOTE@ -o @SHELL_QUOTE@%o@SHELL_QUOTE@" />
   <delegate decode="dvi" command="@SHELL_QUOTE@@DVIDecodeDelegate@@SHELL_QUOTE@ -sstdout=%%stderr -o @SHELL_QUOTE@%o@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@"/>
   <delegate decode="edit" stealth="True" command="@SHELL_QUOTE@@EditorDelegate@@SHELL_QUOTE@ -title @SHELL_QUOTE@Edit Image Comment@SHELL_QUOTE@ -e vi @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
-  <delegate decode="emf" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; @MVDelegate@ @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
+  <delegate decode="emf" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; /opt/bin/mv @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
   <delegate decode="eps" encode="pdf" mode="bi" command="@SHELL_QUOTE@@PSDelegate@@SHELL_QUOTE@ -sstdout=%%stderr -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=500000000 @SHELL_QUOTE@-sDEVICE=@GSPDFDevice@@SHELL_QUOTE@ @SHELL_QUOTE@-sOutputFile=%o@SHELL_QUOTE@ @SHELL_QUOTE@-f%i@SHELL_QUOTE@"/>
   <delegate decode="eps" encode="ps" mode="bi" command="@SHELL_QUOTE@@PSDelegate@@SHELL_QUOTE@ -sstdout=%%stderr -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=500000000 -dAlignToPixels=0 -dGridFitTT=2 @SHELL_QUOTE@-sDEVICE=@GSPSDevice@@SHELL_QUOTE@ @SHELL_QUOTE@-sOutputFile=%o@SHELL_QUOTE@ @SHELL_QUOTE@-f%i@SHELL_QUOTE@"/>
-  <delegate decode="epub" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; @MVDelegate@ @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
-  <delegate decode="hpg" command="@SHELL_QUOTE@@HPGLDecodeDelegate@@SHELL_QUOTE@ -sstdout=%%stderr -m eps -f `basename @SHELL_QUOTE@%o@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@;     @MVDelegate@ -f `basename @SHELL_QUOTE@%o@SHELL_QUOTE@` @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
-  <delegate decode="hpgl" command="@SHELL_QUOTE@@HPGLDecodeDelegate@@SHELL_QUOTE@ -sstdout=%%stderr -m eps -f `basename @SHELL_QUOTE@%o@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@;     @MVDelegate@ -f `basename @SHELL_QUOTE@%o@SHELL_QUOTE@` @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
+  <delegate decode="epub" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; /opt/bin/mv @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
+  <delegate decode="hpg" command="@SHELL_QUOTE@@HPGLDecodeDelegate@@SHELL_QUOTE@ -sstdout=%%stderr -m eps -f `basename @SHELL_QUOTE@%o@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@;     /opt/bin/mv -f `basename @SHELL_QUOTE@%o@SHELL_QUOTE@` @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
+  <delegate decode="hpgl" command="@SHELL_QUOTE@@HPGLDecodeDelegate@@SHELL_QUOTE@ -sstdout=%%stderr -m eps -f `basename @SHELL_QUOTE@%o@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@;     /opt/bin/mv -f `basename @SHELL_QUOTE@%o@SHELL_QUOTE@` @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
   <delegate decode="htm" command="@SHELL_QUOTE@@HTMLDecodeDelegate@@SHELL_QUOTE@ -U -o @SHELL_QUOTE@%o@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@"/>
   <delegate decode="html" command="@SHELL_QUOTE@@HTMLDecodeDelegate@@SHELL_QUOTE@ -U -o @SHELL_QUOTE@%o@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@"/>
   <delegate decode="ilbm" command="@SHELL_QUOTE@@ILBMDecodeDelegate@@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@ &gt; @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
   <delegate decode="jpg" encode="lep" mode="encode" command="@SHELL_QUOTE@@LEPDelegate@@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
-  <delegate decode="jxr" command="@MVDelegate@ @SHELL_QUOTE@%i@SHELL_QUOTE@ @SHELL_QUOTE@%i.jxr@SHELL_QUOTE@; @SHELL_QUOTE@@JXRDecodeDelegate@@SHELL_QUOTE@ -i @SHELL_QUOTE@%i.jxr@SHELL_QUOTE@ -o @SHELL_QUOTE@%o.tiff@SHELL_QUOTE@; @MVDelegate@ @SHELL_QUOTE@%i.jxr@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@; @MVDelegate@ @SHELL_QUOTE@%o.tiff@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
+  <delegate decode="jxr" command="/opt/bin/mv @SHELL_QUOTE@%i@SHELL_QUOTE@ @SHELL_QUOTE@%i.jxr@SHELL_QUOTE@; @SHELL_QUOTE@@JXRDecodeDelegate@@SHELL_QUOTE@ -i @SHELL_QUOTE@%i.jxr@SHELL_QUOTE@ -o @SHELL_QUOTE@%o.tiff@SHELL_QUOTE@; /opt/bin/mv @SHELL_QUOTE@%i.jxr@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@; /opt/bin/mv @SHELL_QUOTE@%o.tiff@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
   <delegate decode="lep" mode="decode" command="@SHELL_QUOTE@@LEPDelegate@@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
   <delegate decode="png" encode="open" spawn="True" command="@SHELL_QUOTE@@OpenDelegate@@SHELL_QUOTE@ -a Preview @SHELL_QUOTE@%i@SHELL_QUOTE@"/>
   <delegate decode="miff" encode="show" spawn="True" command="@SHELL_QUOTE@@DisplayDelegate@@SHELL_QUOTE@ display -immutable -delay 0 -title @SHELL_QUOTE@%M@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@"/>
   <delegate decode="miff" encode="win" stealth="True" spawn="True" command="@SHELL_QUOTE@@DisplayDelegate@@SHELL_QUOTE@ display -immutable -delay 0 -title @SHELL_QUOTE@%M@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@"/>
-  <delegate decode="odg" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; @MVDelegate@ @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
-  <delegate decode="odt" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; @MVDelegate@ @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
+  <delegate decode="odg" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; /opt/bin/mv @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
+  <delegate decode="odt" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; /opt/bin/mv @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
   <delegate decode="pcl:cmyk" stealth="True" command="@SHELL_QUOTE@@PCLDelegate@@SHELL_QUOTE@ -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=500000000 -dAlignToPixels=0 -dGridFitTT=2 @SHELL_QUOTE@-sDEVICE=@PCLCMYKDevice@@SHELL_QUOTE@ -dTextAlphaBits=%u -dGraphicsAlphaBits=%u @SHELL_QUOTE@-r%s@SHELL_QUOTE@ %s @SHELL_QUOTE@-sOutputFile=%s@SHELL_QUOTE@ @SHELL_QUOTE@%s@SHELL_QUOTE@"/>
   <delegate decode="pcl:color" stealth="True" command="@SHELL_QUOTE@@PCLDelegate@@SHELL_QUOTE@ -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=500000000 -dAlignToPixels=0 -dGridFitTT=2 @SHELL_QUOTE@-sDEVICE=@PCLColorDevice@@SHELL_QUOTE@ -dTextAlphaBits=%u -dGraphicsAlphaBits=%u @SHELL_QUOTE@-r%s@SHELL_QUOTE@ %s @SHELL_QUOTE@-sOutputFile=%s@SHELL_QUOTE@ @SHELL_QUOTE@%s@SHELL_QUOTE@"/>
   <delegate decode="pcl:mono" stealth="True" command="@SHELL_QUOTE@@PCLDelegate@@SHELL_QUOTE@ -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=500000000 -dAlignToPixels=0 -dGridFitTT=2 @SHELL_QUOTE@-sDEVICE=@PCLMonoDevice@@SHELL_QUOTE@ -dTextAlphaBits=%u -dGraphicsAlphaBits=%u @SHELL_QUOTE@-r%s@SHELL_QUOTE@ %s @SHELL_QUOTE@-sOutputFile=%s@SHELL_QUOTE@ @SHELL_QUOTE@%s@SHELL_QUOTE@"/>
@@ -98,10 +98,10 @@
   <delegate decode="clipboard" command="@SHELL_QUOTE@@ClipboardDelegate@@SHELL_QUOTE@ -selection clipboard -o &gt; @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
   <delegate decode="png" encode="webp" command="@SHELL_QUOTE@@WebPEncodeDelegate@@SHELL_QUOTE@ -quiet -q %Q @SHELL_QUOTE@%i@SHELL_QUOTE@ -o @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
   <delegate decode="pnm" encode="ilbm" mode="encode" command="@SHELL_QUOTE@@ILBMEncodeDelegate@@SHELL_QUOTE@ -24if @SHELL_QUOTE@%i@SHELL_QUOTE@ &gt; @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
-  <delegate decode="tiff" encode="jxr" command="@MVDelegate@ @SHELL_QUOTE@%i@SHELL_QUOTE@ @SHELL_QUOTE@%i.tiff@SHELL_QUOTE@; @SHELL_QUOTE@@JXREncodeDelegate@@SHELL_QUOTE@ -i @SHELL_QUOTE@%i.tiff@SHELL_QUOTE@ -o @SHELL_QUOTE@%o.jxr@SHELL_QUOTE@ -q %Q; @MVDelegate@ @SHELL_QUOTE@%i.tiff@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@; @MVDelegate@ @SHELL_QUOTE@%o.jxr@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
-  <delegate decode="tiff" encode="wdp" command="@MVDelegate@ @SHELL_QUOTE@%i@SHELL_QUOTE@ @SHELL_QUOTE@%i.tiff@SHELL_QUOTE@; @SHELL_QUOTE@@JXREncodeDelegate@@SHELL_QUOTE@ -i @SHELL_QUOTE@%i.tiff@SHELL_QUOTE@ -o @SHELL_QUOTE@%o.jxr@SHELL_QUOTE@ -q %Q; @MVDelegate@ @SHELL_QUOTE@%i.tiff@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@; @MVDelegate@ @SHELL_QUOTE@%o.jxr@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
-  <delegate decode="ppt" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; @MVDelegate@ @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
-  <delegate decode="pptx" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; @MVDelegate@ @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
+  <delegate decode="tiff" encode="jxr" command="/opt/bin/mv @SHELL_QUOTE@%i@SHELL_QUOTE@ @SHELL_QUOTE@%i.tiff@SHELL_QUOTE@; @SHELL_QUOTE@@JXREncodeDelegate@@SHELL_QUOTE@ -i @SHELL_QUOTE@%i.tiff@SHELL_QUOTE@ -o @SHELL_QUOTE@%o.jxr@SHELL_QUOTE@ -q %Q; /opt/bin/mv @SHELL_QUOTE@%i.tiff@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@; /opt/bin/mv @SHELL_QUOTE@%o.jxr@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
+  <delegate decode="tiff" encode="wdp" command="/opt/bin/mv @SHELL_QUOTE@%i@SHELL_QUOTE@ @SHELL_QUOTE@%i.tiff@SHELL_QUOTE@; @SHELL_QUOTE@@JXREncodeDelegate@@SHELL_QUOTE@ -i @SHELL_QUOTE@%i.tiff@SHELL_QUOTE@ -o @SHELL_QUOTE@%o.jxr@SHELL_QUOTE@ -q %Q; /opt/bin/mv @SHELL_QUOTE@%i.tiff@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@; /opt/bin/mv @SHELL_QUOTE@%o.jxr@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
+  <delegate decode="ppt" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; /opt/bin/mv @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
+  <delegate decode="pptx" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; /opt/bin/mv @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
   <delegate decode="ps" encode="prt" command="@SHELL_QUOTE@@LPRDelegate@@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@"/>
   <delegate decode="ps:alpha" stealth="True" command="@SHELL_QUOTE@@PSDelegate@@SHELL_QUOTE@ -sstdout=%%stderr -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=500000000 -dAlignToPixels=0 -dGridFitTT=2 @SHELL_QUOTE@-sDEVICE=@GSAlphaDevice@@SHELL_QUOTE@ -dTextAlphaBits=%u -dGraphicsAlphaBits=%u @SHELL_QUOTE@-r%s@SHELL_QUOTE@ %s @SHELL_QUOTE@-sOutputFile=%s@SHELL_QUOTE@ @SHELL_QUOTE@-f%s@SHELL_QUOTE@ @SHELL_QUOTE@-f%s@SHELL_QUOTE@"/>
   <delegate decode="ps:cmyk" stealth="True" command="@SHELL_QUOTE@@PSDelegate@@SHELL_QUOTE@ -sstdout=%%stderr -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=500000000 -dAlignToPixels=0 -dGridFitTT=2 @SHELL_QUOTE@-sDEVICE=@GSCMYKDevice@@SHELL_QUOTE@ -dTextAlphaBits=%u -dGraphicsAlphaBits=%u @SHELL_QUOTE@-r%s@SHELL_QUOTE@ %s @SHELL_QUOTE@-sOutputFile=%s@SHELL_QUOTE@ @SHELL_QUOTE@-f%s@SHELL_QUOTE@ @SHELL_QUOTE@-f%s@SHELL_QUOTE@"/>
@@ -115,18 +115,18 @@
   <delegate decode="svg" command="@SHELL_QUOTE@@RSVGDecodeDelegate@@SHELL_QUOTE@ --dpi-x %x --dpi-y %y -o @SHELL_QUOTE@%o@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@"/>
   <!-- Change export-filename to export-png for inkscape < 1.0 -->
   <delegate decode="svg:decode" stealth="True" command="@SHELL_QUOTE@@SVGDecodeDelegate@@SHELL_QUOTE@ @SHELL_QUOTE@%s@SHELL_QUOTE@ @SHELL_QUOTE@--export-filename=%s@SHELL_QUOTE@ @SHELL_QUOTE@--export-dpi=%s@SHELL_QUOTE@ @SHELL_QUOTE@--export-background=%s@SHELL_QUOTE@ @SHELL_QUOTE@--export-background-opacity=%s@SHELL_QUOTE@"/>
-  <delegate decode="tiff" encode="text" command="@SHELL_QUOTE@@TextEncodeDelegate@@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@ @SHELL_QUOTE@%u@SHELL_QUOTE@; @MVDelegate@ @SHELL_QUOTE@%u.txt@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
+  <delegate decode="tiff" encode="text" command="@SHELL_QUOTE@@TextEncodeDelegate@@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@ @SHELL_QUOTE@%u@SHELL_QUOTE@; /opt/bin/mv @SHELL_QUOTE@%u.txt@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
   <delegate decode="tiff" encode="launch" mode="encode" command="@SHELL_QUOTE@@LaunchDelegate@@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@"/>
-  <delegate decode="wdp" command="@MVDelegate@ @SHELL_QUOTE@%i@SHELL_QUOTE@ @SHELL_QUOTE@%i.jxr@SHELL_QUOTE@; @SHELL_QUOTE@@JXRDecodeDelegate@@SHELL_QUOTE@ -i @SHELL_QUOTE@%i.jxr@SHELL_QUOTE@ -o @SHELL_QUOTE@%o.tiff@SHELL_QUOTE@; @MVDelegate@ @SHELL_QUOTE@%i.jxr@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@; @MVDelegate@ @SHELL_QUOTE@%o.tiff@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
+  <delegate decode="wdp" command="/opt/bin/mv @SHELL_QUOTE@%i@SHELL_QUOTE@ @SHELL_QUOTE@%i.jxr@SHELL_QUOTE@; @SHELL_QUOTE@@JXRDecodeDelegate@@SHELL_QUOTE@ -i @SHELL_QUOTE@%i.jxr@SHELL_QUOTE@ -o @SHELL_QUOTE@%o.tiff@SHELL_QUOTE@; /opt/bin/mv @SHELL_QUOTE@%i.jxr@SHELL_QUOTE@ @SHELL_QUOTE@%i@SHELL_QUOTE@; /opt/bin/mv @SHELL_QUOTE@%o.tiff@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
   <delegate decode="webp" command="@SHELL_QUOTE@@WebPDecodeDelegate@@SHELL_QUOTE@ -pam @SHELL_QUOTE@%i@SHELL_QUOTE@ -o @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
-  <delegate decode="wmf" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; @MVDelegate@ @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
-  <delegate decode="xls" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; @MVDelegate@ @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
-  <delegate decode="xlsx" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; @MVDelegate@ @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
+  <delegate decode="wmf" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; /opt/bin/mv @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
+  <delegate decode="xls" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; /opt/bin/mv @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
+  <delegate decode="xlsx" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; /opt/bin/mv @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
   <delegate decode="xps:cmyk" stealth="True" command="@SHELL_QUOTE@@XPSDelegate@@SHELL_QUOTE@ -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=500000000 -dAlignToPixels=0 -dGridFitTT=2 @SHELL_QUOTE@-sDEVICE=@XPSCMYKDevice@@SHELL_QUOTE@ -dTextAlphaBits=%u -dGraphicsAlphaBits=%u @SHELL_QUOTE@-r%s@SHELL_QUOTE@ %s @SHELL_QUOTE@-sOutputFile=%s@SHELL_QUOTE@ @SHELL_QUOTE@%s@SHELL_QUOTE@"/>
   <delegate decode="xps:color" stealth="True" command="@SHELL_QUOTE@@XPSDelegate@@SHELL_QUOTE@ -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=500000000 -dAlignToPixels=0 -dGridFitTT=2 @SHELL_QUOTE@-sDEVICE=@XPSColorDevice@@SHELL_QUOTE@ -dTextAlphaBits=%u -dGraphicsAlphaBits=%u @SHELL_QUOTE@-r%s@SHELL_QUOTE@ %s @SHELL_QUOTE@-sOutputFile=%s@SHELL_QUOTE@ @SHELL_QUOTE@%s@SHELL_QUOTE@"/>
   <delegate decode="xps:mono" stealth="True" command="@SHELL_QUOTE@@XPSDelegate@@SHELL_QUOTE@ -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=500000000 -dAlignToPixels=0 -dGridFitTT=2 @SHELL_QUOTE@-sDEVICE=@XPSMonoDevice@@SHELL_QUOTE@ -dTextAlphaBits=%u -dGraphicsAlphaBits=%u @SHELL_QUOTE@-r%s@SHELL_QUOTE@ %s @SHELL_QUOTE@-sOutputFile=%s@SHELL_QUOTE@ @SHELL_QUOTE@%s@SHELL_QUOTE@"/>
   <delegate decode="video:decode" command="@SHELL_QUOTE@@VIDEODecodeDelegate@@SHELL_QUOTE@ -nostdin -loglevel error -i @SHELL_QUOTE@%s@SHELL_QUOTE@ -an -f rawvideo -y %s @SHELL_QUOTE@%s@SHELL_QUOTE@"/>
   <delegate encode="video:encode" stealth="True" command="@SHELL_QUOTE@@VIDEOEncodeDelegate@@SHELL_QUOTE@ -nostdin -loglevel error -i @SHELL_QUOTE@%s%%d.%s@SHELL_QUOTE@ %s @SHELL_QUOTE@%s.%s@SHELL_QUOTE@"/>
-  <delegate decode="vsd" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; @MVDelegate@ @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
-  <delegate decode="vsdx" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; @MVDelegate@ @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
+  <delegate decode="vsd" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; /opt/bin/mv @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
+  <delegate decode="vsdx" command="@SHELL_QUOTE@@DOCDecodeDelegate@@SHELL_QUOTE@ --convert-to pdf --outdir `dirname @SHELL_QUOTE@%i@SHELL_QUOTE@` @SHELL_QUOTE@%i@SHELL_QUOTE@ 2&gt; @SHELL_QUOTE@%u@SHELL_QUOTE@; /opt/bin/mv @SHELL_QUOTE@%i.pdf@SHELL_QUOTE@ @SHELL_QUOTE@%o@SHELL_QUOTE@"/>
 </delegatemap>
--- a/config/mime.xml
+++ b/config/mime.xml
@@ -213,16 +213,8 @@
   <mime type="application/x-arj" description="ARJ archive" data-type="short" endian="LSB" offset="0" magic="0xea60" priority="50" />
   <mime type="application/x-arj" acronym="ARJ" description="ARJ archive" priority="100" pattern="*.arj" />
   <mime type="application/x-asp" acronym="ASP" description="Active Server Page" priority="100" pattern="*.asp" />
-  <mime type="application/x-awk" description="AWK script" data-type="string" offset="0" magic="#!/bin/gawk" priority="50" />
-  <mime type="application/x-awk" description="AWK script" data-type="string" offset="0" magic="#! /bin/gawk" priority="50" />
-  <mime type="application/x-awk" description="AWK script" data-type="string" offset="0" magic="#!/usr/bin/gawk" priority="50" />
-  <mime type="application/x-awk" description="AWK script" data-type="string" offset="0" magic="#! /usr/bin/gawk" priority="50" />
-  <mime type="application/x-awk" description="AWK script" data-type="string" offset="0" magic="#!/usr/local/bin/gawk" priority="50" />
-  <mime type="application/x-awk" description="AWK script" data-type="string" offset="0" magic="#! /usr/local/bin/gawk" priority="50" />
-  <mime type="application/x-awk" description="AWK script" data-type="string" offset="0" magic="#!/bin/awk" priority="50" />
-  <mime type="application/x-awk" description="AWK script" data-type="string" offset="0" magic="#! /bin/awk" priority="50" />
-  <mime type="application/x-awk" description="AWK script" data-type="string" offset="0" magic="#!/usr/bin/awk" priority="50" />
-  <mime type="application/x-awk" description="AWK script" data-type="string" offset="0" magic="#! /usr/bin/awk" priority="50" />
+  <mime type="application/x-awk" description="AWK script" data-type="string" offset="0" magic="#!/opt/bin/gawk" priority="50" />
+  <mime type="application/x-awk" description="AWK script" data-type="string" offset="0" magic="#!/opt/bin/awk" priority="50" />
   <mime type="application/x-bcpio" acronym="BCPIO" description="BCPIO document" priority="100" pattern="*.bcpio" />
   <mime type="application/x-bittorrent" description="BitTorrent seed file" data-type="string" offset="0" magic="d8:announce" priority="50" />
   <mime type="application/x-bittorrent" description="BitTorrent seed file" priority="100" pattern="*.torrent" />
@@ -264,10 +256,10 @@
   <mime type="application/x-cpio" description="CPIO archive" data-type="short" offset="0" magic="0143561" priority="50" />
   <mime type="application/x-cpio" description="CPIO archive" priority="100" pattern="*.cpio" />
   <mime type="application/x-cpio-compressed" description="CPIO archive (gzip-compressed)" priority="100" pattern="*.cpio.gz" />
-  <mime type="application/x-csh" description="C shell script" data-type="string" offset="1:16" magic="/bin/tcsh" />
-  <mime type="application/x-csh" description="C shell script" data-type="string" offset="1:16" magic="/bin/csh" />
-  <mime type="application/x-csh" description="C shell script" data-type="string" offset="0" magic="#!/usr/bin/env csh" />
-  <mime type="application/x-csh" description="C shell script" data-type="string" offset="0" magic="#!/usr/bin/env tcsh" />
+  <mime type="application/x-csh" description="C shell script" data-type="string" offset="1:16" magic="/opt/bin/tcsh" />
+  <mime type="application/x-csh" description="C shell script" data-type="string" offset="1:16" magic="/opt/bin/csh" />
+  <mime type="application/x-csh" description="C shell script" data-type="string" offset="0" magic="#!/opt/bin/csh" />
+  <mime type="application/x-csh" description="C shell script" data-type="string" offset="0" magic="#!/opt/bin/tcsh" />
   <mime type="application/x-csh" description="C shell script" priority="100" pattern="*.csh" />
   <mime type="application/x-dbf" description="Xbase document" priority="100" pattern="*.dbf" />
   <mime type="application/ecmascript" description="ECMAScript program" priority="100" pattern="*.es" />
@@ -494,9 +486,8 @@
   <mime type="application/x-par2" description="PAR2 Parity File" priority="100" pattern="*.PAR2" />
   <mime type="application/x-par2" description="PAR2 Parity File" priority="100" pattern="*.par2" />
   <mime type="application/x-pef-executable" description="PEF executable" data-type="string" offset="0" magic="Joy!" priority="50" />
-  <mime type="application/x-perl" description="Perl script" data-type="string" offset="0" magic="eval \&quot;exec /usr/local/bin/perl" priority="50" />
-  <mime type="application/x-perl" description="Perl script" data-type="string" offset="1:16" magic="/bin/perl" priority="50" />
-  <mime type="application/x-perl" description="Perl script" data-type="string" offset="1:16" magic="/bin/env perl" priority="50" />
+  <mime type="application/x-perl" description="Perl script" data-type="string" offset="0" magic="eval \&quot;exec /opt/bin/perl" priority="50" />
+  <mime type="application/x-perl" description="Perl script" data-type="string" offset="1:16" magic="/opt/bin/perl" priority="50" />
   <mime type="application/x-perl" description="Perl script" priority="100" pattern="*.pl" />
   <mime type="application/x-perl" description="Perl script" priority="100" pattern="*.pm" />
   <mime type="application/x-perl" description="Perl script" priority="100" pattern="*.al" />
@@ -529,8 +520,7 @@
   <mime type="application/x-reject" description="rejected patch" priority="100" pattern="*.rej" />
   <mime type="application/x-rpm" description="RPM package" data-type="string" offset="0" magic="\xed\xab\xee\xdb" priority="50" />
   <mime type="application/x-rpm" description="RPM package" priority="100" pattern="*.rpm" />
-  <mime type="application/x-ruby" description="Ruby script" data-type="string" offset="1:16" magic="/bin/env ruby" />
-  <mime type="application/x-ruby" description="Ruby script" data-type="string" offset="1:16" magic="/bin/ruby" />
+  <mime type="application/x-ruby" description="Ruby script" data-type="string" offset="1:16" magic="/opt/bin/ruby" />
   <mime type="application/x-ruby" description="Ruby script" priority="100" pattern="*.rb" />
   <mime type="application/x-markaby" description="Markaby script" priority="100" pattern="*.mab" />
   <mime type="application/x-sc" description="SC/Xspread spreadsheet" data-type="string" offset="38" magic="Spreadsheet" priority="50" />
@@ -542,15 +532,15 @@
   <mime type="application/x-sharedlib" description="shared library" data-type="string" offset="0" mask="0xffffffff000000000000000000000000ff" magic="\177ELF \003" priority="50" />
   <mime type="application/x-sharedlib" description="shared library" priority="100" pattern="*.so" />
   <mime type="application/x-shellscript" description="shell script" data-type="string" offset="10" magic="# This is a shell archive" priority="50" />
-  <mime type="application/x-shellscript" description="shell script" data-type="string" offset="1:16" magic="/bin/bash" priority="50" />
-  <mime type="application/x-shellscript" description="shell script" data-type="string" offset="1:16" magic="/bin/nawk" priority="50" />
-  <mime type="application/x-shellscript" description="shell script" data-type="string" offset="1:16" magic="/bin/zsh" priority="50" />
-  <mime type="application/x-shellscript" description="shell script" data-type="string" offset="1:16" magic="/bin/sh" priority="50" />
-  <mime type="application/x-shellscript" description="shell script" data-type="string" offset="1:16" magic="/bin/ksh" priority="50" />
-  <mime type="application/x-shellscript" description="shell script" data-type="string" offset="0" magic="#!/usr/bin/env sh" priority="50" />
-  <mime type="application/x-shellscript" description="shell script" data-type="string" offset="0" magic="#!/usr/bin/env bash" priority="50" />
-  <mime type="application/x-shellscript" description="shell script" data-type="string" offset="0" magic="#!/usr/bin/env zsh" priority="50" />
-  <mime type="application/x-shellscript" description="shell script" data-type="string" offset="0" magic="#!/usr/bin/env ksh" priority="50" />
+  <mime type="application/x-shellscript" description="shell script" data-type="string" offset="1:16" magic="/opt/bin/bash" priority="50" />
+  <mime type="application/x-shellscript" description="shell script" data-type="string" offset="1:16" magic="/opt/bin/nawk" priority="50" />
+  <mime type="application/x-shellscript" description="shell script" data-type="string" offset="1:16" magic="/opt/bin/zsh" priority="50" />
+  <mime type="application/x-shellscript" description="shell script" data-type="string" offset="1:16" magic="/opt/bin/sh" priority="50" />
+  <mime type="application/x-shellscript" description="shell script" data-type="string" offset="1:16" magic="/opt/bin/ksh" priority="50" />
+  <mime type="application/x-shellscript" description="shell script" data-type="string" offset="0" magic="#!/opt/bin/sh" priority="50" />
+  <mime type="application/x-shellscript" description="shell script" data-type="string" offset="0" magic="#!/opt/bin/bash" priority="50" />
+  <mime type="application/x-shellscript" description="shell script" data-type="string" offset="0" magic="#!/opt/bin/zsh" priority="50" />
+  <mime type="application/x-shellscript" description="shell script" data-type="string" offset="0" magic="#!/opt/bin/ksh" priority="50" />
   <mime type="application/x-shellscript" description="shell script" priority="100" pattern="*.sh" />
   <mime type="application/x-shockwave-flash" description="Shockwave Flash file" data-type="string" offset="0" magic="FWS" priority="50" />
   <mime type="application/x-shockwave-flash" description="Shockwave Flash file" data-type="string" offset="0" magic="CWS" priority="50" />
@@ -1014,8 +1004,7 @@
   <mime type="text/x-ldif" acronym="LDIF" description="LDAP Data Interchange Format" priority="100" pattern="*.ldif" />
   <mime type="text/x-literate-haskell" description="Literate Haskell source code" priority="100" pattern="*.lhs" />
   <mime type="text/x-log" description="application log" priority="100" pattern="*.log" />
-  <mime type="text/x-makefile" description="Makefile" data-type="string" offset="0" magic="#!/usr/bin/make" priority="50" />
-  <mime type="text/x-makefile" description="Makefile" data-type="string" offset="0" magic="#! /usr/bin/make" priority="50" />
+  <mime type="text/x-makefile" description="Makefile" data-type="string" offset="0" magic="#!/opt/bin/make" priority="50" />
   <mime type="text/x-makefile" description="Makefile" priority="100" pattern="[Mm]akefile" />
   <mime type="text/x-makefile" description="Makefile" priority="100" pattern="GNUmakefile" />
   <mime type="text/x-moc" acronym="Qt MOC" description="Qt Meta Object Compiler" priority="100" pattern="*.moc" />
@@ -1042,19 +1031,11 @@
   <mime type="text/x-patch" description="differences between files" data-type="string" offset="0" magic="Index:" priority="50" />
   <mime type="text/x-patch" description="differences between files" priority="100" pattern="*.diff" />
   <mime type="text/x-patch" description="differences between files" priority="100" pattern="*.patch" />
-  <mime type="text/x-python" description="Python script" data-type="string" offset="0" magic="#!/bin/python" priority="50" />
-  <mime type="text/x-python" description="Python script" data-type="string" offset="0" magic="#! /bin/python" priority="50" />
-  <mime type="text/x-python" description="Python script" data-type="string" offset="0" magic="eval \&quot;exec /bin/python" priority="50" />
-  <mime type="text/x-python" description="Python script" data-type="string" offset="0" magic="#!/usr/bin/python" priority="50" />
-  <mime type="text/x-python" description="Python script" data-type="string" offset="0" magic="#! /usr/bin/python" priority="50" />
-  <mime type="text/x-python" description="Python script" data-type="string" offset="0" magic="eval \&quot;exec /usr/bin/python" priority="50" />
-  <mime type="text/x-python" description="Python script" data-type="string" offset="0" magic="#!/usr/local/bin/python" priority="50" />
-  <mime type="text/x-python" description="Python script" data-type="string" offset="0" magic="#! /usr/local/bin/python" priority="50" />
-  <mime type="text/x-python" description="Python script" data-type="string" offset="0" magic="eval \&quot;exec /usr/local/bin/python" priority="50" />
-  <mime type="text/x-python" description="Python script" data-type="string" offset="1:16" magic="/bin/env python" priority="50" />
+  <mime type="text/x-python" description="Python script" data-type="string" offset="0" magic="#!/opt/bin/python" priority="50" />
+  <mime type="text/x-python" description="Python script" data-type="string" offset="0" magic="eval \&quot;exec /opt/bin/python" priority="50" />
+  <mime type="text/x-python" description="Python script" data-type="string" offset="1:16" magic="/opt/bin/python" priority="50" />
   <mime type="text/x-python" description="Python script" priority="100" pattern="*.py" />
-  <mime type="text/x-lua" description="Lua script" data-type="string" offset="0" magic="/bin/lua" priority="50" />
-  <mime type="text/x-lua" description="Lua script" data-type="string" offset="0" magic="/bin/env lua" priority="50" />
+  <mime type="text/x-lua" description="Lua script" data-type="string" offset="0" magic="/opt/bin/lua" priority="50" />
   <mime type="text/x-lua" description="Lua script" priority="100" pattern="*.lua" />
   <mime type="text/x-readme" description="README document" priority="100" pattern="README*" />
   <mime type="text/x-readme" description="README document" priority="100" pattern="*.nfo" />
