--- a/DefineOptions.cmake
+++ b/DefineOptions.cmake
@@ -71,7 +71,7 @@ if (WITH_ABI_BREAK)
     set(WITH_SYMBOL_VERSIONING ON)
 endif (WITH_ABI_BREAK)
 
-set(GLOBAL_CONF_DIR "/etc/ssh")
+set(GLOBAL_CONF_DIR "/opt/etc/ssh")
 if (WIN32)
     # Use PROGRAMDATA on Windows
     if (DEFINED ENV{PROGRAMDATA})
--- a/src/bind.c
+++ b/src/bind.c
@@ -236,9 +236,9 @@ int ssh_bind_listen(ssh_bind sshbind)
         sshbind->rsakey == NULL &&
         sshbind->ed25519key == NULL) {
 
-        sshbind->ecdsakey = strdup("/etc/ssh/ssh_host_ecdsa_key");
-        sshbind->rsakey = strdup("/etc/ssh/ssh_host_rsa_key");
-        sshbind->ed25519key = strdup("/etc/ssh/ssh_host_ed25519_key");
+        sshbind->ecdsakey = strdup("/opt/etc/ssh/ssh_host_ecdsa_key");
+        sshbind->rsakey = strdup("/opt/etc/ssh/ssh_host_rsa_key");
+        sshbind->ed25519key = strdup("/opt/etc/ssh/ssh_host_ed25519_key");
     }
 
     if (sshbind->rsa == NULL &&
--- a/src/config.c
+++ b/src/config.c
@@ -650,13 +650,13 @@ ssh_config_make_absolute(ssh_session ses
     /* relative path */
     if (global) {
         /* Parsing global config */
-        outlen = strlen(path) + strlen("/etc/ssh/") + 1;
+        outlen = strlen(path) + strlen("/opt/etc/ssh/") + 1;
         out = malloc(outlen);
         if (out == NULL) {
             ssh_set_error_oom(session);
             return NULL;
         }
-        rv = snprintf(out, outlen, "/etc/ssh/%s", path);
+        rv = snprintf(out, outlen, "/opt/etc/ssh/%s", path);
         if (rv < 1) {
             free(out);
             return NULL;
--- a/src/dh-gex.c
+++ b/src/dh-gex.c
@@ -314,7 +314,7 @@ error:
 
 #ifdef WITH_SERVER
 
-#define MODULI_FILE "/etc/ssh/moduli"
+#define MODULI_FILE "/opt/etc/ssh/moduli"
 /* 2     "Safe" prime; (p-1)/2 is also prime. */
 #define SAFE_PRIME 2
 /* 0x04  Probabilistic Miller-Rabin primality tests. */
--- a/src/socket.c
+++ b/src/socket.c
@@ -1149,11 +1149,11 @@ ssh_execute_command(const char *command,
     if (shell == NULL || shell[0] == '\0') {
         /* Fall back to the /bin/sh only if the bash is not available. But there are
          * issues with dash or whatever people tend to link to /bin/sh */
-        rc = access("/bin/bash", 0);
+        rc = access("$SHELL", 0);
         if (rc != 0) {
-            shell = "/bin/sh";
+            shell = "$SHELL";
         } else {
-            shell = "/bin/bash";
+            shell = "$SHELL";
         }
     }
     args[0] = shell;
