--- a/contrib/syslog-ng-debun
+++ b/contrib/syslog-ng-debun
@@ -48,12 +48,12 @@ netstatnlp () { netstat -nlp ; }
 netstatlunp () { netstat -lunp ; }
 netstatpunt () { netstat -punt ; }
 netstatpn () { netstat -pn ; }
-netstatsu="netstat -su"
-binprefix=/opt/syslog-ng
+netstatsu="netstat -u"
+binprefix=/opt
 user_binprefix=
-absscldirs="/usr/share/syslog-ng/include/scl"
+absscldirs="/opt/share/syslog-ng/include/scl"
 relscldirs="share/include/scl share/syslog-ng/include/scl"
-workdir=/tmp
+workdir=/opt/tmp
 lsof="lsof -p"
 no_lsof_fallback() { echo "No lsof in path."; }
 pscmd="ps auxwwwwwf"
@@ -65,7 +65,7 @@ dfk="df -k"
 dfh="df -h"
 dfi="df -i"
 duks="du -ks"
-grepq="fgrep -q"
+grepq="grep -F -q"
 lddcmd="ldd"
 topcmd () { top -b -n 1 -c >"${1}"; }
 opensslcmd="openssl"
@@ -90,7 +90,7 @@ getsyslogpids () { pidof syslog-ng ; }
 os_hash_helper () { find . '!' \( -name debun.manifest -o -name syslog-ng.debun.txt \) -type f -print0 | xargs -0 md5sum ; }
 dfk_parser () { tail -1 | while read FS ALL USED AVAIL UPERC MP; do if echo ${AVAIL} | ${grepq} '%'; then echo ${USED} ; else echo ${AVAIL} ; fi done ; }
 trace="strace -s256 -ff -ttT -f"
-initfile="/etc/init.d/syslog-ng"
+initfile="/opt/etc/init.d/S01syslog-ng"
 service_stop="${initfile} stop"
 service_start="${initfile} start"
 service_status="${initfile} status"
@@ -113,12 +113,12 @@ Usage: syslog-ng-debun [OPTIONS]
 General Options:
   -r		Run actual information gathering
   -h		Show this help page
-  -R [dir]	Syslog-ng's alternate install dir, instead of /opt/syslog-ng
+  -R [dir]	Syslog-ng's alternate install dir, instead of /opt
   -W [dir]	Work dir, where debug bundle will be placed
   -l		"light" collect: Don't get data, which may disturb your sense about
 		privacy, like process tree, fstab, etc. If you use with -d, then it
 		will also enlighten that's params: $default_ldebug_params
-  -K		Include the private keys in the /etc/syslog-ng or /opt/syslog-ng/etc directory
+  -K		Include the private keys in the /opt/etc/syslog-ng or /opt/etc directory
 
 Debug mode options:
   -d		Debug with params: $default_debug_params
@@ -469,7 +469,7 @@ acquire_network_info () {
 		printf "Failed\n"
 	fi
 	printf "Getting DNS resolution-related information: "
-	[ -f /etc/nsswitch.conf ] && cp /etc/nsswitch.conf ${tmpdir}/sys.nsswitch.conf
+	[ -f /opt/etc/nsswitch.conf ] && cp /opt/etc/nsswitch.conf ${tmpdir}/sys.nsswitch.conf
 	[ -f /etc/resolv.conf ] && cp /etc/resolv.conf ${tmpdir}/sys.resolv.conf
 	[ -f /etc/hosts ] && cp /etc/hosts ${tmpdir}/sys.hosts
 	printf "Done\n"
@@ -1035,11 +1035,11 @@ fhs_apply_binprefix_override () {
 }
 
 fhs_set_linux () {
-	fhs_set_paths_from_prefix /usr
+	fhs_set_paths_from_prefix /opt
 
-	confdir=/etc/syslog-ng
-	vardir=/var/lib/syslog-ng
-	piddir=/var/lib/syslog-ng
+	confdir=/opt/etc
+	vardir=/opt/var
+	piddir=/opt/var/run/syslog-ng
 }
 
 fhs_set_unix () {
@@ -1356,7 +1356,7 @@ detect_env_linux () {
 		dist="SUSE LINUX"
 	else
 		echo "Unknown or unsupported Linux distribution!"
-		cat /etc/*release /etc/*version >${tmpdir}/sys.linux.os-release 2>/dev/null
+		cat /opt/etc/*release /etc/*release /etc/*version >${tmpdir}/sys.linux.os-release 2>/dev/null
 	fi
 }
 
@@ -1383,6 +1383,9 @@ detect_env () {
 			elif [ -d /etc/syslog-ng/ ]; then
 				syslogfhs=linux
 				echo "Linux-type FHS detected"
+			elif [ -x /opt/bin/loggen ]; then
+				syslogfhs=linux
+				echo "Linux-type FHS detected"
 			else
 				syslogfhs=unknown
 				confdir=/nonexistent
