# https://github.com/curl/curl/pull/15026

---

--- a/lib/curl_trc.h
+++ b/lib/curl_trc.h
@@ -126,6 +126,12 @@ extern struct curl_trc_feat Curl_trc_fea
 void Curl_trc_ws(struct Curl_easy *data,
                  const char *fmt, ...) CURL_PRINTF(2, 3);
 #endif
+#if defined(USE_WEBSOCKETS) && !defined(CURL_DISABLE_HTTP)
+static void Curl_trc_ws(struct Curl_easy *data, const char *fmt, ...)
+{
+  (void)data; (void)fmt;
+}
+#endif
 
 #define CURL_TRC_M_is_verbose(data) \
   Curl_trc_ft_is_verbose(data, &Curl_trc_feat_multi)
