--- a/lib/defines.h
+++ b/lib/defines.h
@@ -84,19 +84,19 @@
 #define WEEK (7*DAY)
 
 #ifndef PASSWD_FILE
-#define PASSWD_FILE "/etc/passwd"
+#define PASSWD_FILE "/opt/etc/passwd"
 #endif
 
 #ifndef GROUP_FILE
-#define GROUP_FILE "/etc/group"
+#define GROUP_FILE "/opt/etc/group"
 #endif
 
 #ifndef SUBUID_FILE
-#define SUBUID_FILE "/etc/subuid"
+#define SUBUID_FILE "/opt/etc/subuid"
 #endif
 
 #ifndef SUBGID_FILE
-#define SUBGID_FILE "/etc/subgid"
+#define SUBGID_FILE "/opt/etc/subgid"
 #endif
 
 /*
