--- a/src/fccfg.c
+++ b/src/fccfg.c
@@ -2566,6 +2566,8 @@ FcConfigXdgDataDirs (void)
 	 *
 	 * If $XDG_DATA_DIRS is either not set or empty, a value equal to /usr/local/share/:/usr/share/ should be used.
 	 */
+	FcStrSetAdd (ret, (const FcChar8 *)"/opt/usr/share");
+	FcStrSetAdd (ret, (const FcChar8 *)"/opt/share");
 	FcStrSetAdd (ret, (const FcChar8 *)"/usr/local/share");
 	FcStrSetAdd (ret, (const FcChar8 *)"/usr/share");
     }
